client/my-sites/plugins/related-plugins/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { PluginPrice } from 'calypso/my-sites/plugins/plugin-price';
+import PreinstalledPremiumPluginPriceDisplay from 'calypso/my-sites/plugins/plugin-price/preinstalled-premium-plugin-price-display';
+import { useSelector } from 'calypso/state';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import './style.scss';
+import { getFirstCategoryFromTags } from '../categories/use-categories';
+
+type RelatedPluginProps = {
+	slug: string;
+	size: number;
+	seeAllLink?: string;
+	options: {
+		enabled?: boolean;
+		staleTime?: number;
+		refetchOnMount?: boolean;
+	};
+};
+
+function RelatedPlugins( { slug, size = 4, seeAllLink, options }: RelatedPluginProps ) {
+	const translate = useTranslate();
+	const { data: relatedPlugins } = useGetRelatedPlugins( slug, size, { ...options } );
+
+	return (
+		<div className="related-plugins">
+			<div className="related-plugins__header">
+				<h2>{ translate( 'Related plugins' ) }</h2>
+				{ seeAllLink && (
+					<Button borderless primary>
+						<span>{ translate( 'See all' ) }</span>
+						<Gridicon icon="chevron-right" />
+					</Button>
+				) }
+			</div>
+			<div className="related-plugins__list">
+				{ relatedPlugins &&
+					relatedPlugins.map( ( plugin: RelatedPlugin ) => (
+						<RelatedPluginCard key={ plugin.slug } plugin={ plugin } />
+					) ) }
+			</div>
+		</div>
+	);
+}
+
+function RelatedPluginCard( { plugin }: { plugin: RelatedPlugin } ): JSX.Element {
+	const translate = useTranslate();
+	const selectedSite = useSelector( getSelectedSite );
+
+	const pluginLink = useMemo( () => {
+		let url = '/plugins/' + plugin.slug;
+		if ( selectedSite ) {
+			url += '/' + selectedSite.slug;
+		}
+		return url;
+	}, [ plugin.slug, selectedSite ] );
+
+	return (
+		<a className="related-plugins-item" href={ pluginLink }>
+			<PluginIcon image={ plugin.icon } className="related-plugins-item__icon" />
+			<div className="related-plugins-item__info">
+				<h3 className="related-plugins-item__title">{ plugin.title }</h3>
+				<div className="related-plugins-item__excerpt">{ plugin.excerpt }</div>
+				<div className="related-plugins-item__details">
+					<PluginPrice plugin={ plugin } billingPeriod={ IntervalLength.MONTHLY }>
+						{ ( { isFetching, price, period } ) => {
+							if ( isFetching ) {
+								return '...';
+							}
+
+							if ( price ) {
+								return (
+									<PreinstalledPremiumPluginPriceDisplay
+										className="related-plugins-item__price-interval"
+										period={ period }
+										pluginSlug={ plugin.slug }
+										price={ price }
+									/>
+								);
+							}
+
+							return <>{ translate( 'Free' ) }</>;
+						} }
+					</PluginPrice>
+					{ plugin.categories.length > 0 && (
+						<span className="related-plugins-item__category">
+							{ getFirstCategoryFromTags( plugin.categories ) }
+						</span>
+					) }
+				</div>
+			</div>
+		</a>
+	);
+}
+
+export default RelatedPlugins;
```

Comment:
Oops, this is left from the component I copied when adding this! :D AFAIK, there's no benefit to default exports, I prefer named exports wherever possible because of the reasons you've mentioned above!

Reply:
Added it in 11f2fc91113f6d6f8891ec341fc26ae282196f6e.